src/app/Shared/Model/Isolate.model.ts

```diff
@@ -165,7 +165,7 @@ export interface IsolateApiInterface {
                 value: string;
                 active: boolean;
             };
-        };
+        }; */
```

Comment:
See comment in review of previous commit

Reply:
done.